daemon/containerd/image_list_test.go

```diff
@@ -251,3 +306,59 @@ func (s *blobsDirContentStore) Info(ctx context.Context, dgst digest.Digest) (co
 func (s *blobsDirContentStore) Update(ctx context.Context, info content.Info, fieldpaths ...string) (content.Info, error) {
 	return content.Info{}, fmt.Errorf("read-only")
 }
+
+// delayedStore is a content store wrapper that adds a random delay to all
```

Comment:
The delay is not random though, it's defined once when the store is created

Reply:
Thanks, good catch! I made it random at first but then figured out that it's actually not a good idea.

Edited the comment.